docs/Kubernetes/kubernetes.md

```diff
@@ -0,0 +1,70 @@
+### Introduction
+Note: Not for production use!!
+This guide with provide detail on how to deploy the AMTCloudToolkit services in Microsoft(C) Azure(R).
```

Comment:
"This guide explains how to deploy the AMTCloudToolkit services in Microsoft Azure*."

I'm putting the "*" there because that is the way that Intel websites like software.intel.com handle that particular 3rd party product. 

It just appears simpler this way. Now is AMTCloudToolkit the name of something, a set of services? You're not referring to Open AMT Cloud Toolkit in general, right? I just want to make sure that I'm clear on that.